src/Templates/src/templates/maui-mobile/MauiProgram.cs

```diff
@@ -20,9 +20,11 @@ public static MauiApp CreateMauiApp()
 			.ConfigureSyncfusionToolkit()
 			.ConfigureMauiHandlers(handlers =>
 			{
+//-:cnd:noEmit
 #if IOS || MACCATALYST
```

Comment:
The tests are failing because I think the CV2 is not in the blank maui app, should it be used in all templates, or just when using the sample content?